### PR TITLE
docs: src/man/man9 - formatting

### PR DESCRIPTION
**Pull request by smoe**
Thinking to have completed the review of the formatting in src/man/man9.

### CODE REVIEW

**Comment by petterreinholdtsen** [comment]
The github CI seem to fail on this change.  I guess it need to be fixed before it can be merged.

File: docs/src/man/man9/hal_bb_gpio.9.adoc [hunk 1]
```diff
@@ -18,10 +18,16 @@ Linux.
...
+<<<<<<< HEAD
 **bb_gpio.userled*_N_ bit in +
 **bb_gpio.userled*_N_*-invert* bit in::
   The associated LED is lit if *userled*_N_ xor *userled*_N_-invert is
   TRUE.
+=======
+**bb_gpio.userled**_N_ bit in +
+**bb_gpio.userled**_N_**-invert** bit in::
+  The associated LED is lit if *userled*_N_ xor *userled*_N_-invert is TRUE.
+>>>>>>> e0fabd0926 (docs: src/man/man9 - formatting)
```
Comment (hansu) [inline docs/src/man/man9/hal_bb_gpio.9.adoc:30]: here

**Review by hansu — CHANGES_REQUESTED** [review]
Thanks a lot for fixing this!

I marked the spots where the build fails...

File: docs/src/man/man9/encoder_ratio.9.adoc [hunk 2]
```diff
@@ -6,42 +6,44 @@ encoder_ratio - an electronic gear to synchronize two axes
 
 == SYNOPSIS
 
+<<<<<<< HEAD
 **loadrt encoder_ratio [num_chan=**_num_** | names=**_name1_**[,**_name2..._**]]**
+=======
+*loadrt encoder_ratio* [ **num_chan=**_num_ | **names=**_name1_[,_name2_...] ]
+>>>>>>> e0fabd0926 (docs: src/man/man9 - formatting)
```
Comment (hansu) [inline docs/src/man/man9/encoder_ratio.9.adoc:13]: here

File: docs/src/man/man9/hal_parport.9.adoc [hunk 3]
```diff
@@ -7,8 +7,12 @@ parallel ports.
 
 == SYNOPSIS
 
+<<<<<<< HEAD
 **loadrt hal_parport cfg="__port_addr__ [_type_] [[__port_addr__ [_type_]
 ...]"**
+=======
+**loadrt hal_parport cfg="**__port_addr__ [_type_] [ [__port_addr__ [__type__] .. .]**"**
+>>>>>>> e0fabd0926 (docs: src/man/man9 - formatting)
```
Comment (hansu) [inline docs/src/man/man9/hal_parport.9.adoc:15]: here

File: docs/src/man/man9/hal_parport.9.adoc [hunk 4]
```diff
@@ -174,23 +170,35 @@ hal_parport.c). The ports are numbered starting at zero.
...
   describes the parallel ports to be used, and (optionally) their types.
   If the configuration string does not describe at least one port, it is
+<<<<<<< HEAD
   an error. +
   *loadrt hal_parport cfg="port [type] [port [type] ...]"*
+=======
+  an error.
+
+**loadrt hal_parport cfg="**__port__ [__type__] [__port__ [__type__] ...]**"**
+>>>>>>> e0fabd0926 (docs: src/man/man9 - formatting)
```
Comment (hansu) [inline docs/src/man/man9/hal_parport.9.adoc:180]: here

File: docs/src/man/man9/hal_parport.9.adoc [hunk 5]
```diff
@@ -174,23 +170,35 @@ hal_parport.c). The ports are numbered starting at zero.
...
 *Using the Port Address*::
+<<<<<<< HEAD
   Instead, the port address may be specified using the hex notation 0x
   then the address. +
   *loadrt hal_parport cfg="0x378"*
+=======
+  Instead, the port address may be specified using the hex notation 0x then the address.
+
+*loadrt hal_parport cfg="0x378"*
+>>>>>>> e0fabd0926 (docs: src/man/man9 - formatting)
```
Comment (hansu) [inline docs/src/man/man9/hal_parport.9.adoc:201]: here

File: docs/src/man/man9/hal_parport.9.adoc [hunk 6]
```diff
@@ -234,12 +242,16 @@ time.
 == SEE ALSO
 
 Parallel Port Driver (Hardware Drivers Section of LinuxCNC Docs),
+<<<<<<< HEAD
 PCI Parallel Port Example(Hardware Examples Section of LinuxCNC Docs)
+=======
+PCI Parallel Port Example (Hardware Examples Section of LinuxCNC Docs)
+>>>>>>> e0fabd0926 (docs: src/man/man9 - formatting)
```
Comment (hansu) [inline docs/src/man/man9/hal_parport.9.adoc:249]: here

**Comment by smoe** [comment]
I have fixed those oversights (and one more :-) ) and then rebased to a refreshed master. 

There is some trouble ahead since for some unknown reason we now get an increased sensitivity on line breaks again. I have a hunch that the parsing of lines starting with something `*bold*`, i.e. with an asterisk, are somehow mistaken for something else that triggers mid-sentence line breaks now. We have some 600 translations now suffering under that symptom.

**Comment by hansu** [comment]
And it seems that we hove some limitations now - e.g. bold tex in the NAME section is still ignored. See counter.9.adoc.

I continued based on your work with further fine-tuning (
https://github.com/hansu/linuxcnc/commit/aafbd8e5be26b4771db2193424cdeb63fd96de5e). I will commit that after this fix is merged.

**Pull request merged**